app/questionnaire/rules.py

```diff
@@ -52,7 +50,7 @@ def evaluate_goto(goto_rule, metadata, answer_store, group_instance):
     return True
 
 
-def evaluate_repeat(repeat_rule, answer_store, current_path):
+def evaluate_repeat(repeat_rule, answer_store, answer_on_path):
```

Comment:
Related to comment above, it would be clearer if `answer_on_path` was renamed `answer_ids_on_path`